java/client/src/test/java/glide/api/RedisClientCreateTest.java

```diff
@@ -78,8 +84,8 @@ public void createClient_errorOnConnectionThrowsExecutionException() {
     connectToRedisFuture.completeExceptionally(exception);
     RedisClientConfiguration config = RedisClientConfiguration.builder().build();
 
-    when(connectionManager.connectToRedis(eq(config))).thenReturn(connectToRedisFuture);
-    mockedClient.when(() -> CreateClient(config)).thenCallRealMethod();
+    when(connectionManager.connectToRedis(any())).thenReturn(connectToRedisFuture);
+    mockedClient.when(() -> CreateClient(any())).thenCallRealMethod();
```

Comment:
can we revert this change?  This would mean that we know which prototype message gets sent.